.circleci/config.yml

```diff
@@ -1,119 +1,14 @@
-      - restore_cache:
-          keys:
-            - was-registrar-v1-{{ checksum "Gemfile.lock" }}
-            - was-registrar-v1-
-      - run:
-          name: Bundle Install
-          command: bundle check || bundle install
-      # Store bundle cache
-      - save_cache:
-          key: was-registrar-v1-{{ checksum "Gemfile.lock" }}
-          paths:
-            - vendor/bundle
-      # Only necessary if app uses webpacker or yarn in some other way
-      - restore_cache:
-          keys:
-            - rails-demo-yarn-{{ checksum "yarn.lock" }}
-            - rails-demo-yarn-
-      - run:
-          name: Yarn Install
-          command: yarn install --cache-folder ~/.cache/yarn
-      # Store yarn / webpacker cache
-      - save_cache:
-          key: rails-demo-yarn-{{ checksum "yarn.lock" }}
-          paths:
-            - ~/.cache/yarn
-      - run:
-          name: compile assets
-          command: NODE_ENV=test bundle exec rails assets:precompile
-      # Copy packs into the workspace
-      - persist_to_workspace:
-          root: ~/project
-          paths: '*'
-  test:
-    docker:
-      - *default_docker_ruby_executor
-      - *postgres
-    steps:
-      - attach_workspace:
-          at: '~/project'
-      - restore_cache:
-          keys:
-            - was-registrar-v1-{{ checksum "Gemfile.lock" }}
-            - was-registrar-v1-
-      - run:
-          name: Install bundler
-          command: gem install bundler
-      - run:
-          name: Bundle Install
-          command: bundle check || bundle install
-      - restore_cache:
-          keys:
-            - rails-demo-yarn-{{ checksum "yarn.lock" }}
-            - rails-demo-yarn-
-      - run:
-          name: Rubocop
-          command: bundle exec rubocop
-      - run:
-          name: Wait for DB
-          command: dockerize -wait tcp://localhost:5432 -timeout 1m
-      - run:
-          name: Database setup
-          command: bundle exec rails db:schema:load --trace
-      - run:
-          name: Setup Code Climate test-reporter
-          command: |
-            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-            chmod +x ./cc-test-reporter
-            ./cc-test-reporter before-build
-      - run:
-          name: rspec
-          command: bundle exec rspec
-      - run:
-          name: upload test coverage report to Code Climate
-          command: ./cc-test-reporter after-build --coverage-input-type simplecov --exit-code $?
+version: 2.1
+orbs:
+  ruby-rails: sul-dlss/ruby-rails@2.0.0
 workflows:
-  version: 2
-  build_and_test:
+  build:
     jobs:
-      - build
-      - test:
-          requires:
-            - build
+      - ruby-rails/lint:
+          name: lint
+      - ruby-rails/test-rails:
+          name: test
+          before-test:
+            - run:
+                name: compile assets
+                command: NODE_ENV=test bundle exec rails assets:precompile
```

Comment:
I don't think NODE_ENV needs to be set here, does it?

Reply:
Apparently not.